src/hooks/useAsyncTasks/useAsyncTasks.ts

```diff
@@ -22,10 +34,10 @@ const useAsyncTasks = <R>(tasks: Task<R>[], options: Options<R>) => {
     data: null,
     error: null,
   });
+  const [resetTrigger, setResetTrigger] = useState(0); // reset 호출 시 useEffect 재실행을 위한 상태
```

Comment:
```suggestion
const [state, dispatch] = useReducer(reducer<R>, {
  isLoading: false,
  resetTrigger: 0,
  data: null,
  error: null,
});
```

[P2] resetTrigger를 reducer로 관리할 수 있을까요?!

Reply:
`resetTrigger` 상태는 useEffect 훅을 특수한 상황에서만 재실행하기 위한 트릭으로 사용했습니다!
다만 `resetTrigger` 상태를 리듀서로 함께 관리하게 되면, 특정 상황(예: 리셋)에서만 useEffect훅을 재실행하는 것이 어려울 수 있습니다.

Reply:
저는 `state.resetTrigger`를 useEffect의 의존성 배열에 넣는 걸 생각했는데 혹시 이럴 경우에 의도한 대로 reset 로직이 작동하지 않을까요?

(`ACTION_TYPES.RESET` 디스패치에서 resetTrigger 플래그를 관리하는 것을 전제로 말씀드린 내용입니다!)

Reply:
[commit] [cdb703c](https://github.com/frontend-opensource-project/use-react-hooks/pull/92/commits/cdb703c69039239053ebb28a7a9f6951aa7f220d)

@bicochan 아 이해했습니다 네 맞습니다 😃
의존성 배열에 특정 상태만 전달하여 useEffect가 의도한 대로 재실행되도록 수정했습니다. 좋은 피드백 감사합니다!